tvtk/tests/test_tvtk.py

```diff
@@ -200,7 +200,7 @@ def test_object_cache(self):
         # The test sometimes fails as VTK seems to generate objects with the
         # same memory address and hash, we try to force it to allocate more
         # objects so as to not end up reusing the same address and hash.
-        junk = [vtk.vtkConeSource() for i in range(5)]
+        junk = [tvtk.ConeSource() for i in range(50)]
```

Comment:
This is related to an issue with deleting vtk objects, which has been fixed. See: http://review.source.kitware.com/#/c/15095/. This test is flaky on VTK 5.x, not on 6.x. Not necessary to add an issue ticket for this.

Other reference: https://github.com/enthought/mayavi/pull/138.